plugins/nf-wave/src/main/io/seqera/wave/plugin/WaveClient.groovy

```diff
@@ -125,7 +125,7 @@ class WaveClient {
         this.fusion = new FusionConfig(session.config.fusion as Map ?: Collections.emptyMap(), SysEnv.get())
         this.tower = new TowerConfig(session.config.tower as Map ?: Collections.emptyMap(), SysEnv.get())
         this.awsFargate = WaveFactory.isAwsBatchFargateMode(session.config)
-        this.s5cmdConfigUrl = session.config.navigate('wave.s5cmdConfigUrl') as URL
+        this.s5cmdConfigUrl = parseUrl(session.config.navigate('wave.s5cmdConfigUrl') as String)
```

Comment:
Not sure this is related to this PR

Reply:
You are right, I dragged it from your original PR, but it is out of scope here. It is factored in a single commit, happy to take it out

Reply:
OK, reverting here, opened dedicated PR #4707